include/ttmlir/Dialect/TTNN/Transforms/Passes.td

```diff
@@ -86,4 +86,35 @@ def TTNNCreateInputGenerators: Pass<"ttnn-create-input-gens", "::mlir::ModuleOp"
   }];
 }
 
+def TTNNModifySignaturesForDylib: Pass<"ttnn-modify-signatures-for-dylib", "::mlir::ModuleOp"> {
+  let summary = "Modify signatures of the functions for dylib path.";
+  let description = [{
+    This pass modifies signatures of forward functions so that they're
+    compatible for dylib path. Essentially, input tensors are packed into a
```

Comment:
Do you mean here for dylib lookup?

Reply:
Yeah, that was a poor wording choice, I'll change it.

By dylib path, I wanted to say "pipeline" that ends up with dylib, as opposed to flatbuffer or raw C++.